releases/latest/edge/mlflow/bundle.yaml

```
@@ -0,0 +1,25 @@
+bundle: kubernetes
+name: mlflow
+applications:
+  mlflow-minio:
+    charm: minio
+    channel: ckf-1.7/edge
+    scale: 1
+    trust: true
+    _github_repo_name: minio-operator
+  mysql-k8s:
```

```suggestion
  mlflow-mysql:
```

and potentially we need to deploy mysql controller